app/controllers/user_controller.rb

```diff
@@ -29,8 +29,6 @@ def trn_redirect_path
     end
 
     def user_params
-      params.require(:user).permit(
-        :school_name
-      )
+      params.require(:user).permit()
```

Comment:
What does `permit` without any arguments achieve here? My understanding was `permit!` would permit anything but this permits nothing? I may be wrong.